diff --git a/test/network.bats b/test/network.bats
index d4fd2ee21..d3808aaad 100644
--- a/test/network.bats
+++ b/test/network.bats
@@ -185,17 +185,3 @@ function teardown() {
 	num_allocated=$(ls /var/lib/cni/networks/crionet_test_args | wc -l)
 	[[ "${num_allocated}" == "0" ]]
 }
-
-@test "Clean up network if pod sandbox fails after plugin success" {
-	start_crio "" "" "" "" "prepare_plugin_test_args_network_conf_malformed_result"
-
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -ne 0 ]
-
-	# ensure that the server cleaned up sandbox networking if the sandbox
-	# failed during network setup after the CNI plugin itself succeeded
-	rm -f /var/lib/cni/networks/crionet_test_args/last_reserved_ip
-	num_allocated=$(ls /var/lib/cni/networks/crionet_test_args | wc -l)
-	[[ "${num_allocated}" == "0" ]]
-}
